eva_submission/eload_validation.py

```diff
@@ -353,6 +358,71 @@ def _collect_validation_workflow_results(self, output_dir):
             })
         self.eload_cfg.set('validation', 'assembly_check', 'pass', value=total_error == 0)
 
+    def _detect_structural_variant(self):
+        # Initializing the list of boolean values to store the status of the VCFs denoting whether it has a SV or not
+        has_sv = []
+        vcf_files = self._get_vcf_files()
+        for vcf_file in vcf_files:
+            has_sv_per_vcf = self._detect_structural_variant_from_variant_lines(vcf_file)
+            if not has_sv_per_vcf:
+                vcf_name = os.path.basename(vcf_file)
+                validator_output_file = os.path.join(self._get_dir('vcf_check'), vcf_name + '.vcf_validator.txt')
+                has_sv_per_vcf = self._detect_structural_variant_from_validator_output(validator_output_file)
+            has_sv.append(has_sv_per_vcf)
+        return has_sv
```

Comment:
You returning `has_sv` but you're not using the return in `validate`

Reply:
I have addressed this now